File: app/src/main/java/com/originb/inkwisenote/modules/repositories/HandwrittenNoteWithImage.java
package com.originb.inkwisenote.modules.repositories;

import android.graphics.Bitmap;
import com.originb.inkwisenote.data.entities.handwrittennotedata.HandwrittenNoteEntity;

import java.util.Optional;

public class HandwrittenNoteWithImage {
    public HandwrittenNoteEntity handwrittenNoteEntity;
    public Optional<Bitmap> noteImage;
}

File: app/src/main/java/com/originb/inkwisenote2/modules/handwrittennotes/data/HandwrittenNoteWithImage.kt
package com.originb.inkwisenote2.modules.handwrittennotes.data

import android.graphics.Bitmap
import java.util.*

class HandwrittenNoteWithImage {
    @JvmField
    var handwrittenNoteEntity: HandwrittenNoteEntity? = null
    @JvmField
    var noteImage: Optional<Bitmap?>? = null
}
